tests/test.py

```diff
@@ -74,6 +74,17 @@ def test_stopping_vm(self):
         self.assertEqual(0, process.returncode)
         self.assertEqual('Stopped', output.rstrip())
 
+    def test_swapspace(self):
+        ''' Test if swap space is available on restart '''
+        cmd = self.bin_dir + "minishift start"
+        self.execute_test({ 'cmd': cmd })
+    
+        # Check swap space
+        cmd = self.bin_dir + "minishift ssh \"echo `free | tail -n 1 | awk '{print $2}'`\""
```

Comment:
@budhrg we can change the other code as well, but as part of a later refactor. WDYT?